app/data/doc-collections/cadence__styleguide.json

```diff
@@ -0,0 +1,11 @@
+{
+  "source": {
+    "owner": "onflow",
+    "name": "cadence-style-guide",
+    "branch": "main",
+    "rootPath": "docs/"
+  },
+  "manifest": {
```

Comment:
Your flow-docs.json is the "manifest" for this doc collection. 
A display name is required here. A user-defined display name will overwrite this if present.